e2e_tests/application_test.py

```diff
@@ -38,36 +38,51 @@ def test_create_individual_application():
 
 
 def create_business_application():
-    request = CreateBusinessApplicationRequest(
-        name="Acme Inc.",
-        address=Address("1600 Pennsylvania Avenue Northwest", "Washington", "CA", "20500", "US"),
-        phone=Phone("1", "9294723497"), state_of_incorporation="CA", entity_type="Corporation", ein="123456789",
-        officer=Officer(full_name=FullName("Jone", "Doe"), date_of_birth=date.today() - timedelta(days=20 * 365),
-                           address=Address("950 Allerton Street", "Redwood City", "CA", "94063", "US"),
-                           phone=Phone("1", "2025550108"), email="jone.doe@unit-finance.com", ssn="123456789"),
-        contact=BusinessContact(full_name=FullName("Jone", "Doe"), email="jone.doe@unit-finance.com", phone=Phone("1", "2025550108")),
-        beneficial_owners=[
-            BeneficialOwner(
+    try:
+        request = CreateBusinessApplicationRequest(
```

Comment:
Am I understanding that in CreateBusinessApplicationRequest there are only formatting changes?

Reply:
small change to do benOff optional